docs/ref/contrib/humanize.txt

```diff
@@ -56,8 +58,8 @@ You can pass in either an integer or a string representation of an integer.
 ``intword``
 ===========
 
-Converts a large integer to a friendly text representation. Works best for
-numbers over 1 million.
+Converts a large integer or a float to a friendly text representation. Works
```

Comment:
I don't think floats are acceptable here. In my testing, `'1000000.1'` gives the same value back rather than "1 million".